Assets/Scripts/RedRunner/Characters/RedCharacter.cs

```diff
@@ -601,7 +600,8 @@ public override void Die ( bool blood )
 					Destroy ( particle.gameObject, particle.main.duration );
 				}
 				CameraController.Singleton.fastMove = true;
-			}
+                RoundsManager.Local.CmdDeactivateSelf();
```

Comment:
This isn;t particularly safe bc i believe die can be called multiple times if you got multiple spikes, for example. 
also [since you didnt uses my Action >:) ], you're failing to decrement players if they get to the finish.

Accepting for now to unblock others. ill submit a followup to fix these edge cases